PythonRegression/tests/features/machine1/1_api_tests.feature

```diff
@@ -38,56 +53,174 @@ Feature: Test API calls on Machine 1
 		|numberOfSentTransactions			|
 		|connectionType						|	 
 		
-		
+	Scenario: Add and Remove Neighbors
+	    Adds nodeB as a neighbor to nodeA, and then removes it.
+
+	    Given "addNeighbors" is called on "nodeA" with:
+	    |keys       |values                 |type           |
+        |uris       |nodeB                  |nodeAddress    |
+
+	    Then a response with the following is returned:
+	    |keys                               |
+	    |addedNeighbors                     |
+	    |duration                           |
+
+	    When "removeNeighbors" is called on "nodeA" with:
+        |keys       |values                 |type           |
+        |uris       |nodeB                  |nodeAddress    |
+
+
+	    Then a response with the following is returned:
+	    |keys                               |
+	    |duration                           |
+	    |removedNeighbors                   |
+
+
 	Scenario: GetTips is called
-		Given "getTips" is called on "nodeA"
+		Given "getTips" is called on "nodeA" with:
+		|keys       |values                 |type           |
+
 		Then a response with the following is returned:
 		|keys 								|
 		|hashes								|
 		|duration							|
 
-	
-	Scenario Outline: GetTrytes is called 
+
+
+    #Values can be found in util/static_vals.py
+   	Scenario Outline: GetTrytes is called
 		Given getTrytes is called with the hash <hash>
 		Then the response should be equal to <trytes>
 		
 		Examples:
-			|hash 		| trytes 		| 
-			|TEST_HASH	| TEST_TRYTES	|
-	
-		
+			|hash 		| trytes 		    |
+			|TEST_HASH	| TEST_TRYTES	    |
+
+
+
 	Scenario: GetTransactionsToApprove is called
-		Given "getTransactionsToApprove" is called on "nodeA"
+		Given "getTransactionsToApprove" is called on "nodeA" with:
+		|keys       |values                 |type           |
+        |depth      |3                      |int            |
+
 		Then a response with the following is returned: 
 		|keys								|
-		|trunkTransaction					|
 		|branchTransaction					|
 		|duration							|
-		
+		|trunkTransaction					|
+
+
+    Scenario: CheckConsistency is called
+        Given "checkConsistency" is called on "nodeA" with:
+        |keys           |values             |type           |
+        |tails          |TEST_HASH          |staticList     |
+
+        Then a response with the following is returned:
```

Comment:
Since we have a static db, maybe you can check the returned values and not just the keys

Reply:
For this test I will even find a case that returns `true` and a case that returns `false`.